app/Http/Controllers/Forum/TopicController.php

```diff
@@ -62,6 +62,12 @@ public function index(Request $request, $forum, $topic)
             $page = $this->post->getPage(min(2147483647, (int) $request->get('p')), $topic->id, $perPage);
         }
 
+        // show posts of last page if page parameter is higher than pages count
```

Comment:
widzę że ten kod poszedł nie do tego pull requesta co trzeba :) ale to nic.

Reply:
Chyba z branch'y powinienem robić PR'y :)